src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp

```diff
@@ -160,6 +160,48 @@ class jit_exp_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const;
 };
 
+class jit_maximum_emitter : public jit_emitter {
```

Comment:
Minor: This PR seems to denpend on a commit in your another PR. Perhaps let's add:
```
Prerequisites:
https://github.com/openvinotoolkit/openvino/pull/24165
```
in the description section of this PR.

Reply:
done, thanks